Sources/SwiftProtobuf/AnyUnpackError.swift

```diff
@@ -14,23 +14,29 @@
 
 import Foundation
 
-/// Any objects can be parsed from Protobuf Binary, Protobuf Text, or JSON.
-/// The contents are not parsed immediately; the raw data is held in the Any
-/// object until you `unpack()` it into a message.  At this time, any
+/// Describes errors that can occur when unpacking an `Any` message.
+///
+/// `Any` messages can be decoded from protobuf binary, text format, or JSON.
+/// The contents are not parsed immediately; the raw data is held in the `Any`
+/// message until you `unpack()` it into a message.  At this time, any
 /// error can occur that might have occurred from a regular decoding
 /// operation.  In addition, there are a number of other errors that are
-/// possible, involving the structure of the Any object itself.
+/// possible, involving the structure of the `Any` object itself.
 public enum AnyUnpackError: Error {
-  /// The `urlType` field in the Any object did not match the message type
+  /// The `typeURL` field in the `Any` message did not match the message type
   /// provided to the `unpack()` method.
   case typeMismatch
+
   /// Well-known types being decoded from JSON must have only two
-  /// fields:  the `@type` field and a `value` field containing
+  /// fields: the `@type` field and a `value` field containing
   /// the specialized JSON coding of the well-known type.
   case malformedWellKnownTypeJSON
-  /// There was something else wrong...
+
+  /// The `Any` message was malformed in some other way not covered by the other
+  /// error cases.
   case malformedAnyField
-  /// Decoding JSON or Text format requires the message type
+
+  /// Decoding JSON or text format requires the message type
   /// to have been compiled with textual field names.
   case missingFieldNames
```

Comment:
I think this case can be removed; it's never actually used in the code.  (Missing field names is a JSON- or Text-level error and throws either `JSONDecodingError.missingFieldNames` or `TextFormatDecodingError.missingFieldNames` respectively.)